packages/devtools/lib/src/timeline/flame_chart.dart

```diff
@@ -256,9 +258,7 @@ class FlameChartItem {
       ..backgroundColor =
           colorToCss(selected ? selectedFlameChartItemColor : backgroundColor)
       ..border = selected ? '1px solid' : 'none'
-      ..borderColor = selected
-          ? colorToCss(const Color(0x5A1B1F23))
-          : colorToCss(const Color(0x231B1F23));
+      ..borderColor = colorToCss(const Color(0x5A1B1F23));
```

Comment:
Use themed color to clarify this color is intentionally the same in light and dark mode.